client/components/site-selector/add-site.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button, Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent, useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import isJetpackCloud from 'calypso/lib/jetpack/is-jetpack-cloud';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import useOnboardingUrl from './use-onboarding-url';
+
+const SiteSelectorAddSite: FunctionComponent = () => {
+	const onboardingUrl = useOnboardingUrl();
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const recordAddNewSite = useCallback( () => {
+		const event = isJetpackCloud()
```

Comment:
I was trying to think of what to do when `isLoadingExperimentAssignment` returns true. It seems pretty unlikely that the user would be so fast that they could quickly find the "Add new site" button before the assignment was loaded. But if they did, would it be ok to just stop the click from working? But then when they click it again it magically works?

I'm worried `<a href="?ref=calypso-selector">` will reload the page or something.

```suggestion
	const recordAddNewSite = useCallback( ( e: React.MouseEvent< HTMLAnchorElement > ) => {
		if ( ! onboardingUrl ) {
			e.preventDefault();
			return;
		}
		const event = isJetpackCloud()
```

Reply:
TIL about the `busy` prop. This looks like it's exactly what we need 👍

Reply:
nice that works really well, thanks